dsc/tests/dsc_args.tests.ps1

```diff
@@ -180,9 +180,9 @@ resources:
         $LASTEXITCODE | Should -Be 0
     }
 
-    It 'resource tracing shows up' {
+    It 'resource tracing shows up' -Skip:(!$IsWindows) {
```

Comment:
Maybe (as a separate PR) we should have our dsctest resource emit output to stderr and test against that in addition to the ps-adapter

Reply:
Yes, that would make tests cross-platform.